### ISSUE TITLE
nginx reverse proxy - download links drop port

### ISSUE THREAD

**Issue by Quiksmage** [Jackett/Jackett#2177]
Hi, when setting up nginx on a non-standard port, I'm having issues downloading the torrents after searching

for reference, here is my nginx conf for jackett specifically

```
#Jackett
        location /jackett {
                proxy_pass http://192.168.1.61:10005;
                proxy_set_header Host $http_host;
                proxy_set_header X-Real_IP $remote_addr;
                proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
                proxy_set_header X-Forwarded-Proto $scheme;
                proxy_set_header X-Forwarded-Ssl on;
                proxy_set_header Upgrade $http_upgrade;
                proxy_set_header Connection "Upgrade";
                proxy_redirect default;
                }
```

I access my domain on a port other than 443
https://domain.com:9920/jackett

Everything appears to work fine except downloading torrents.
The link for the torrent drops the port and looks similar to the following
https://domain.com/jackett/dl/iptorrents/?jackett_apikey=[apikey]&path=[lots of stuff here]

what can I do to have it retain the port 9920 on download links?

Thanks in advance!

**Comment by kaso17** [comment]
Please try setting a fixed Host header (I suspect the port is missing from $http_host)
`proxy_set_header Host "domain.com:9920";`

**Comment by Quiksmage** [comment]
just tried this now and it's the same.
download links specifically are dropping the port.

**Comment by Quiksmage** [comment]
![image](https://user-images.githubusercontent.com/12449533/33096835-3403c75e-cebd-11e7-9435-59e59861806d.png)

**Comment by kaso17** [comment]
probably a jackett bug then, will have a look tomorrow

**Comment by Quiksmage** [comment]
Thanks!

**Comment by kaso17** [comment]
fixed via 72df2c1545c9989523f73bc4d88cd5526a24cda1

**Comment by Quiksmage** [comment]
Thanks, this is working now :)
